app/hub/Views/UpgradePlanView/UpgradePlanView.jsx

```diff
@@ -0,0 +1,958 @@
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import React, { useRef } from 'react';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import globals from '../../../../src/classes/Globals';
+
+/**
+ * A React class component for rendering the Home View
+ * @return {JSX} JSX for rendering the Home View of the Hub app
+ * @memberof HubComponents
+ */
+const UpgradePlanView = (props) => {
+	const {
+		protection_level,
+		show_yearly_prices,
+		user,
+		actions,
+	} = props;
+
+	const {
+		toggleMonthlyYearlyPrices,
+		setBasicProtection,
+		setPlusProtection,
+		setPremiumProtection
+	} = actions;
+
+	const isPlus = (user && user.plusAccess) || false;
+	const isPremium = (user && user.premiumAccess) || false;
+
+	const BASIC = 'BASIC';
+	const PLUS = 'PLUS';
+	const PREMIUM = 'PREMIUM';
+
+	const sliderClassNames = ClassNames('switch-check', {
+		checked: show_yearly_prices
+	});
+	const tabsTitleBlueClassNames = ClassNames('tabs-title tabs-title-blue', {
+		'is-active': protection_level === BASIC
+	});
+	const tabsTitleGoldClassNames = ClassNames('tabs-title tabs-title-gold', {
+		'is-active': protection_level === PLUS
+	});
+	const tabsTitlePurpleClassNames = ClassNames('tabs-title tabs-title-purple', {
+		'is-active': protection_level === PREMIUM
+	});
+	const monthlyToggleLabel = ClassNames('toggle-label', {
+		active: !show_yearly_prices
+	});
+	const yearlyToggleLabel = ClassNames('toggle-label', {
+		active: show_yearly_prices
+	});
+
+	// Clicking arrow scrolls to table
+	const comparisonTableRef = useRef(null);
+	const scrollToComparisonTable = () => {
+		comparisonTableRef.current.scrollIntoView({ behavior: 'smooth' });
+	};
+	// Clicking arrow scrolls to table for mobile view
+	const mobileComparisonTableRef = useRef(null);
+	const scrollToMobileComparisonTable = () => {
+		mobileComparisonTableRef.current.scrollIntoView({ behavior: 'smooth' });
+	};
+
+	// UTM params
+	const signedIn = +!!user;
+	const subscriptionType = () => {
+		if (isPremium) return 'PREMIUM';
+		if (isPlus) return 'SUPPORTER';
+		return '-1';
+	};
+	const subscriptionBillingType = show_yearly_prices ? 'yearly' : 'monthly';
+
+	// Query Param to show monthly/yearly pricing in checkout web
+	const interval = show_yearly_prices ? 'year' : 'month';
+
+	const plusCheckoutLink = utm_params => (show_yearly_prices ? `${globals.CHECKOUT_BASE_URL}/plus?interval=${interval}&${utm_params}` : `${globals.CHECKOUT_BASE_URL}/plus?interval=${interval}&${utm_params}`);
+	const premiumCheckoutLink = utm_params => (show_yearly_prices ? `${globals.CHECKOUT_BASE_URL}/premium?interval=${interval}&${utm_params}` : `${globals.CHECKOUT_BASE_URL}/premium?interval=${interval}&${utm_params}`);
```

Comment:
I don't think you need the ternaries, right? Looks like you are returning the same thing regardless of the value of `show_yearly_prices`

Reply:
Oh it looks like I am returning the same thing. Removed ternary!